script/pop.js

```diff
@@ -0,0 +1,124 @@
+    seeSourceLink: '#',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    src: './image/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+  {
+    title: 'Tonic',
+    src: './image/Snapshoot-3Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories',
+    src: './image/Snapshoot-4Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+];
+
+const popButtons = document.getElementsByClassName('view');
+const work = document.querySelector('.cards-radious');
+const popup = document.createElement('section');
+popup.classList.add('popup-body');
+const overlay = document.createElement('div');
+overlay.setAttribute('id', 'overlay');
+const { body } = document;
+
+function popupDisplay(cardNum) {
+  body.style.overflowY = ('hidden');
+  overlay.style.display = ('block');
+  popup.style.display = ('block');
+  work.appendChild(overlay);
+  work.appendChild(popup);
+  popup.innerHTML = `
+        <div class="tonic-popup">
+          <h2>${projects[cardNum].title}</h2>
+          <i id="closeBtn" class="fas fa-times" alt='#'></i>
+        </div>
+        <ul class="list-popup">
+          <h2 class="canopy">CANOPY</h2>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; Back End Dev</li>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; 2015</li>
+        </ul>
+        <div class="card-img">
+          <img src="${projects[cardNum].src}" alt="Project-preview-1">
+        </div>
+        <div class="media-img">
+        <img src="${projects[cardNum].src}" alt="Project-preview-1">
+      </div>
+      <div class='flex'>
```

Comment:
- [ ] OPTIONAL: I suggest you to give fixed space for both sides of the section, currently in screen size of 1025 the right side is taking too much space: 
![image](https://user-images.githubusercontent.com/77038610/152639296-c4e2246f-d531-49f2-a01c-ec4486471859.png)